components/credentials/src/main/java/org/cloudfoundry/credhub/generators/SignedCertificateGenerator.java

```diff
@@ -93,11 +93,24 @@ private X509Certificate getSignedByIssuer(
 
     final BigInteger certificateSerialNumber = serialNumberGenerator.generate();
 
+    Date nvb = Date.from(now);
+    Data nva = Date.from(now.plus(Duration.ofDays(params.getDuration())));
+    if (issuerCertificate != null) {
```

Comment:
Could you add unit test coverage for this new logic? I'm assuming we didn't already have good assertions for it, even though the tests hit this code.